integration-tests/pom.xml

```diff
@@ -78,6 +78,12 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.tachyonproject</groupId>
+      <artifactId>tachyon-underfs-local</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This is for `TestUtils` in `tachyon-servers`. We should remove this dependency. A TODO added as comment.